<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KexiProjectStorageTypeSelectionPage</class>
 <widget class="QWidget" name="KexiProjectStorageTypeSelectionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>194</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="_2" columnstretch="0">
     <item row="0" column="0">
      <widget class="KexiCommandLinkButton" name="btn_file">
       <property name="text">
        <string comment="New Project Stored in File">File</string>
       </property>
       <property name="description">
        <string>New Project Stored in File</string>
       </property>
       <property name="arrowVisible" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="KexiCommandLinkButton" name="btn_server">
       <property name="text">
        <string comment="New Project Stored on Database Server">Server</string>
       </property>
       <property name="description">
        <string>New Project Stored on Database Server</string>
       </property>
       <property name="arrowVisible" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>58</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KexiCommandLinkButton</class>
   <extends>QCommandLinkButton</extends>
   <header location="global">kexiutils/KexiCommandLinkButton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">k3listview.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
